benchmark/src/main/scala/mesosphere/marathon/core/experimental/repository/TemplateRepositoryBenchmark.scala

```diff
@@ -9,15 +9,15 @@ import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.{Sink, Source}
 import com.typesafe.scalalogging.StrictLogging
 import mesosphere.marathon.core.storage.zookeeper.{AsyncCuratorBuilderFactory, AsyncCuratorBuilderSettings, ZooKeeperPersistenceStore}
-import mesosphere.marathon.experimental.repository.TemplateRepositoryLike.Template
 import mesosphere.marathon.experimental.repository.TemplateRepository
+import mesosphere.marathon.experimental.repository.TemplateRepositoryLike.Template
 import mesosphere.marathon.metrics.Metrics
 import mesosphere.marathon.metrics.dummy.DummyMetrics
-import mesosphere.marathon.state.{AppDefinition, PathId}
+import mesosphere.marathon.state.{AbsolutePathId, AppDefinition}
 import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory}
 import org.apache.curator.retry.BoundedExponentialBackoffRetry
 import org.apache.curator.x.async.api.CreateOption
-import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Fork, Level, Measurement, Mode, OutputTimeUnit, Param, Scope, State, TearDown, Warmup}
+import org.openjdk.jmh.annotations._
```

Comment:
Yeah, let's just keep this as is

```suggestion
import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Fork, Level, Measurement, Mode, OutputTimeUnit, Param, Scope, State, TearDown, Warmup}
```